### PR TITLE
Batch Repayment strategy - New feature

### PR DESCRIPTION
**Pull request by mohitsinha**
## Description
Describe the changes made and why they were made. Ignore if these details are present on the associated Jira ticket

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [x] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [x] API documentation at https://github.com/apache/fineract/blob/develop/api-docs/apiLive.htm has been updated with details of any API changes.

- [x] Integration tests have been created/updated for verifying the changes made.

- [x] All Integrations tests are passing with the new commits.

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

**Comment by mohitsinha** [comment]
@vorburger Can you tag someone to look into this?

**Comment by vorburger** [comment]
@awasum @angelboxes @muarachmann @vishwasbabu anyone willing to review this one?

PS: @mohitsinha what I do in general is just check the git log and tag [recently active people](https://github.com/apache/fineract/commits/develop)... ;-)

File: fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java [hunk 1]
```diff
@@ -92,6 +92,8 @@ private void init() {
         this.commandStrategies.put(CommandContext.resource("loans\\/\\d+\\/charges").method("POST").build(), "createChargeCommandStrategy");
         this.commandStrategies
                 .put(CommandContext.resource("loans\\/\\d+\\/charges").method("GET").build(), "collectChargesCommandStrategy");
+        this.commandStrategies
+                .put(CommandContext.resource("loans\\/\\d+\\/transactions").method("POST").build(), "repaymentCommandStrategy");
```
Comment (vishwasbabu) [inline fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java]: Please name the new repayment strategy inline with the existing names. Ex : "repayLoanCommandStrategy"

**Review by vishwasbabu — COMMENTED** [review]
@mohitsinha : Other than the comment regarding naming conventions, the pr looks good. Can you also please address the remaining items from the PR checklist before the same can be merged?

1) Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

2) API documentation at https://github.com/apache/fineract/blob/develop/api-docs/apiLive.htm has been updated with details of any API changes.

3) Integration tests have been created/updated for verifying the changes made.

File: fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java (hunk 1 above)
Comment (mohitsinha) [reply fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java]: I have made those changes. Thanks

**Comment by mohitsinha** [comment]
https://jira.apache.org/jira/browse/FINERACT-767
I created a new task on jira for this.

No new Api's have been created hence, no need to update the documentation.

**Comment by vishwasbabu** [comment]
API docs list details of all available commands (see https://demo.openmf.org/api-docs/apiLive.htm#batch_api), the same would need to be updated with details of the newly added command.

**Comment by mohitsinha** [comment]
I have made those changes. 
Thanks @vishwasbabu

**Pull request merged**